package.json

```diff
@@ -56,7 +56,7 @@
     "react-transform-hmr": "^1.0.4",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
-    "webpack": "^1.14.0",
+    "webpack": "2",
```

Comment:
Yeah, I watched with curiosity when I last `yarn upgrade`'d… it just left the `package.json` alone and updated the `yarn.lock` file.